mkt/developers/tests/test_forms.py

```diff
@@ -53,6 +54,9 @@ def test_preview_size(self):
             copyfileobj(open(get_image_path(name)), f)
         assert form.is_valid(), form.errors
         form.save(self.addon)
+        # Since the task is a post-request-task and we are outside the normal
+        # request-response cycle, manually send the tasks.
+        post_request_task._send_tasks()
```

Comment:
Apparently - any view tests should already be fine, since the `request_finished` signal (which `post_request_tasks` hooks up to) should be sent when we call a view through the test client.